torch/linalg/__init__.py

```diff
@@ -106,22 +116,22 @@
 
 .. seealso::
 
-        :func:`torch.linalg.pinv` computes the Moore-Penrose pseudoinverse of matrices of
-        any shape.
+        :func:`torch.linalg.pinv` computes the pseudoinverse (Moore-Penrose inverse) of matrices
+        of any shape.
 
-        :func:torch.linalg.solve computes ``input.inv() @ other``.
+        :func:`torch.linalg.solve` computes :attr:`A`\ `.pinv() @ \ `:attr:`B`.
         It is always prefered to use :func:`torch.linalg.solve` when possible, as it is
         faster and more stable than computing the inverse and then multiplying.
 
 Args:
-    input (Tensor): tensor of shape `(*, n, n)` where `*` is zero or more batch dimensions
+    A (Tensor): tensor of shape `(*, n, n)` where `*` is zero or more batch dimensions
                     consisting of invertible matrices.
 
 Keyword args:
     out (Tensor, optional): output tensor. Ignored if `None`. Default: `None`.
 
 Raises:
-    RuntimeError: if the :attr:`input` matrix or any matrix in the batch :attr:`input` is not invertible
+    RuntimeError: if the :attr:`A` matrix or any matrix in the batch :attr:`A` is not invertible.
```

Comment:
"if the matrix :attr:\`A\` or any matrix in a batch of matrices :attr:\`A\` is not invertible."